cmd/podman/system/service.go

```diff
@@ -52,8 +52,9 @@ func init() {
 
 	flags := srvCmd.Flags()
 
+	cfg := registry.PodmanConfig()
 	timeFlagName := "time"
-	flags.Int64VarP(&srvArgs.Timeout, timeFlagName, "t", 5, "Time until the service session expires in seconds.  Use 0 to disable the timeout")
```

Comment:
Does this actually work? This flag requires an argument, and it's not enabled unless explicitly specified. The default is always going to be overridden?

Reply:
Yes that is how it works. The same logic is used for many flags in other commands.

Reply:
So - how does this help us, then? The unit file is going to have to specify `-t <value>` which overrides the default we're setting here, so `containers.conf` is getting ignored.

Reply:
Which unit includes the `-t` flag?
```
$ systemctl cat podman.service 
# /usr/lib/systemd/system/podman.service
[Unit]
Description=Podman API Service
Requires=podman.socket
After=podman.socket
Documentation=man:podman-system-service(1)
StartLimitIntervalSec=0

[Service]
Type=exec
KillMode=process
Environment=LOGGING="--log-level=info"
ExecStart=/usr/bin/podman $LOGGING system service

[Install]
WantedBy=multi-user.target
```

Reply:
https://github.com/containers/podman/blob/main/contrib/systemd/system/podman.service

Reply:
OH - having a timeout is the default, you have to explicitly opt out with `-t 0`.

OK, this works, then. Nevermind.